### ISSUE TITLE
Extensions README does not show HTML

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#7033]
### Bug description

Unsure if this is intentional or not, but you cannot show HTML in the extensions readme:

![Screenshot 2024-04-30 at 10 40 43 AM](https://github.com/containers/podman-desktop/assets/6422176/8949961c-4120-4a13-b621-4600c96893c5)

### Operating system

mac m1

### Installation Method

None

### Version

1.10.0

### Steps to reproduce

Put HTML in an extensions README.

It's not rendered correctly.

**Comment by deboer-tim** [comment]
It would be nice to have this, but just confirming that we've changed the readme for now so this is not blocking.

**Comment by odockal** [comment]
Isn't it supposed to be markdown format instead of html?

**Comment by deboer-tim** [comment]
Yes, but Github allows you to embed simple html for some things that are difficult in markdown (e.g. in this case including a scaled and centred image), so it's not uncommon for little bits to slip into a readme.

**Comment by deboer-tim** [comment]
Still an issue - e.g. see Headlamp in the catalog.

**Comment by cdrage** [comment]
Still an issue

**Comment by cdrage** [comment]
still an issue
